### ISSUE TITLE
Fix Odd Brackets Large Text Issue

### ISSUE THREAD

**Issue by KeniMorri** [humphd/brackets#108]
In relation to LARGE Text, brackets acts oddly in relation to where the user's Cursor Highlights.

Setup:
Chrome,
Thimble in windowed mode

Steps:
1. Load thimble,
2. Change Size to Large,
3. Press the area below the paragraph,
From here it branches into 2 possible routes:

4a. The cursor correctly gets placed on line 11(the last empty line),
5a. Reload Page, 
6a. Press the area below the paragraph,
After this, the cursor should be placed in the wrong place, and not line 11

4b. The cursor is in the wrong place, and not line 11

Conclusion thus far: It appears that brackets thinks we're currently editing a normal text document instead of a large text. It is possible to fix the cursor by pressing down, then up after step 4b/6a

There is presently no reliable method to get this error to present itself, sometimes the steps outlined above will not work as expected.

Example of Error:
https://www.youtube.com/watch?v=Ba7H14cFcY0&feature=youtu.be

**Comment by humphd** [comment]
Is this a duplicate of #105?  If so, please close.  If not, please add details.

**Comment by humphd** [comment]
Why are we persisting the state of the editor?  Why not have a reload just reset everything?  If it's putting things in the filesystem, note that we are formatting it on every reload.

**Comment by AJDBenner** [comment]
This PR should fix this issue https://github.com/sedge/thimble.webmaker.org/pull/20

**Comment by gideonthomas** [comment]
Fixed with efeafe76d1ea5ce7556b6eb3055892f5c87f1104
